### PR TITLE
Composition and left factor for epis/acyclic maps

### PR DESCRIPTION
**Pull request by tomdjong**

### CODE REVIEW

File: src/foundation/epimorphisms.lagda.md [hunk 1]
```diff
@@ -182,6 +182,25 @@ If the map `f : A → B` is epi, then its codiagonal is an equivalence.
...
+```agda
+module _
+  {l1 l2 l3 : Level} {A : UU l1} {B : UU l2} {C : UU l3}
+  (f : A → B) (g : B → C)
+  where
+
+  is-epimorphism-comp :
+    is-epimorphism f → is-epimorphism g → is-epimorphism (g ∘ f)
+  is-epimorphism-comp ef eg X =
+    is-emb-comp (precomp f X) (precomp g X) (ef X) (eg X)
```
Comment (fredrik-bakke) [inline src/foundation/epimorphisms.lagda.md:196]: I'm not sure how consistent our library is on this point, but I believe the standard ordering for composition lemmas like this is the opposite one. I.e. the argument order should be `g f eg ef`. Did you base your ordering on another definition?

Comment (EgbertRijke) [reply src/foundation/epimorphisms.lagda.md:196]: Good point! The library should be fairly consistent with this:

- composition is written with composition order, i.e. `g o f`
- concatenation is written in diagrammating order, i.e. `f . g`

Thanks for noticing this!

Comment (tomdjong) [reply src/foundation/epimorphisms.lagda.md]: Thanks! This was an oversight by me. Please check if things are right now.

File: src/synthetic-homotopy-theory/acyclic-maps.lagda.md [hunk 2]
```diff
@@ -209,6 +209,34 @@ module _
...
+        ( is-epimorphism-is-acyclic-map g ag)
+        ( is-epimorphism-is-acyclic-map f af))
+
+  is-acyclic-map-left-factor :
+    is-acyclic-map f → is-acyclic-map (g ∘ f) → is-acyclic-map g
+  is-acyclic-map-left-factor af ac =
+    is-acyclic-map-is-epimorphism g
+      ( is-epimorphism-left-factor g f
+        ( is-epimorphism-is-acyclic-map f af)
+        ( is-epimorphism-is-acyclic-map (g ∘ f) ac))
```
Comment (fredrik-bakke) [inline src/synthetic-homotopy-theory/acyclic-maps.lagda.md:237]: Apologies, following the example of `is-equiv-left-factor` this one too should have the arguments swapped:
```suggestion
  is-acyclic-map-left-factor :
    is-acyclic-map (g ∘ f) → is-acyclic-map f → is-acyclic-map g
  is-acyclic-map-left-factor ac af =
    is-acyclic-map-is-epimorphism g
      ( is-epimorphism-left-factor g f
        ( is-epimorphism-is-acyclic-map f af)
        ( is-epimorphism-is-acyclic-map (g ∘ f) ac))
```

Comment (tomdjong) [reply src/synthetic-homotopy-theory/acyclic-maps.lagda.md]: Ah, got it, cheers! I've also updated `is-epimorphism-left-factor` accordingly.

**Pull request merged**